internal/dag/builder_test.go

```diff
@@ -5479,6 +5550,57 @@ func TestDAGInsert(t *testing.T) {
 				},
 			),
 		},
+		// This tests what happens when you build with an ingressroute and a HTTPProxy
```

Comment:
suggestion: try the imperative mood

// Assert that a route in IngressRoute takes precedence over Ingress.

Reply:
Hmm, good idea, that's much more concise. I'll change this, and then merge.